src/pages/settings.html

```diff
@@ -0,0 +1,93 @@
+<!DOCTYPE html>
+<meta charset="UTF-8" />
+<meta http-equiv="X-UA-Compatible" content="IE=edge" />
+<meta name="viewport" content="width=device-width, initial-scale=1.0" />
+<style>
+  @import url("browser://theme/index.css");
+  @import url("browser://theme/vars.css");
+  @import url("browser://theme/settings.css");
+</style>
+<title>Settings - Peersky Browser</title>
+
+<main class="settings-container">
+  <section id="general-settings" class="settings-section">
+    <h1>General Settings</h1>
+    
+    <div class="setting-group">
+      <label for="search-engine">Search Engine</label>
+      <select id="search-engine" class="settings-select">
+        <option value="duckduckgo">DuckDuckGo</option>
+        <option value="ecosia">Ecosia</option>
+        <option value="google">Google</option>
+        <option value="bing">Bing</option>
+        <option value="brave">Brave Search</option>
```

Comment:
We don't need these. Let's have DuckDuckGo (default), Ecosia, and Startpage for now.

These are also worth exploring-
https://mastodon.nz/@weekendspy/113922740975728963